### ISSUE TITLE
Document possible value of status somewhere

### ISSUE THREAD

**Issue by hadley** [r-lib/revdepcheck#195]
Because I keep getting confused

**Comment by gaborcsardi** [comment]
You mean the internal status, supplied by rcmdcheck? I.e. the `t-`, `t+`, `+`, etc.?

**Comment by hadley** [comment]
Yeah

**Comment by gaborcsardi** [comment]
Yeah, I messed this up a lot of times. Added some comments here: https://github.com/r-lib/rcmdcheck/blob/master/R/comparison.R#L19-L35

But I'll also document this is in the rcmdcheck docs, and add a link to that from revdepcheck.
